app/assets/stylesheets/fonts.css

```diff
@@ -0,0 +1,17 @@
+@font-face {
+  font-family: 'Inter var';
```

Comment:
Do these _need_ to be single quoted? If not, let's update this file to only use double quotes.

Reply:
done